spark2/src/main/java/org/apache/iceberg/spark/source/Reader.java

```diff
@@ -136,7 +143,7 @@
     if (io.getValue() instanceof HadoopFileIO) {
       String fsscheme = "no_exist";
       try {
-        Configuration conf = SparkSession.active().sessionState().newHadoopConf();
+        Configuration conf = new Configuration(activeSparkSession().sessionState().newHadoopConf());
```

Comment:
Many changes in this file seems to be copied over from LinkedIn's fork which are not relevant to apache/iceberg. Can you remove these?

The PR over linkedin/iceberg does not have these changes either. So not sure how those were copied over.